build.gradle

```diff
@@ -8,25 +8,24 @@ buildscript {
         // Android jetpack
         archLifecycleVersion = "2.1.0"
         roomVersion = "2.2.5"
-        appCompat = "1.1.0"
+        appCompat = "1.2.0"
         sqliteVersion = "2.1.0"
         lifecycleLiveData = "2.2.0"
         biometricVersion="1.0.1"
 
         // Kotlin
-        kotlinVersion = "1.3.72"
-        coroutinesVersion = "1.3.2"
+        kotlinVersion = "1.4.10"
```

Comment:
I observed this causes the already commited 636ec200ac61a408 code style change

Reply:
What does that mean?